stubs/colorama/colorama/winterm.pyi

```diff
@@ -1,22 +1,23 @@
 import sys
+from typing import ClassVar
 
 if sys.platform == "win32":
     from . import win32
 
     class WinColor:
-        BLACK: int = ...
-        BLUE: int = ...
-        GREEN: int = ...
-        CYAN: int = ...
-        RED: int = ...
-        MAGENTA: int = ...
-        YELLOW: int = ...
-        GREY: int = ...
+        BLACK: ClassVar[int]
+        BLUE: ClassVar[int]
+        GREEN: ClassVar[int]
+        CYAN: ClassVar[int]
+        RED: ClassVar[int]
+        MAGENTA: ClassVar[int]
+        YELLOW: ClassVar[int]
+        GREY: ClassVar[int]
 
     class WinStyle:
-        NORMAL: int = ...
-        BRIGHT: int = ...
-        BRIGHT_BACKGROUND: int = ...
+        NORMAL: ClassVar[int]
+        BRIGHT: ClassVar[int]
+        BRIGHT_BACKGROUND: ClassVar[int]
```

Comment:
These all look like they could _probably_ be `Final`, but that could cause false-positive errors if I'm incorrect about that, and I'm not confident enough to risk it

Reply:
Seems pretty clear these should be final. They're primitive enums.